s3tests/readwrite.py

```diff
@@ -195,7 +195,7 @@ def main():
             warmup_pool = gevent.pool.Pool(size=100)
             for file_name in file_names:
                 fp = next(files)
-                warmup_pool.spawn_link_exception(
+                warmup_pool.spawn(
```

Comment:
@theanalyst 
Yes there need to have some exception handling code - this patch was just a quick trying